### PR TITLE
Add details in LSP1 page of Contracts section

### PR DESCRIPTION
**Pull request by b00ste**
# What does this PR introduce?

In this PR the following changes are made:

- Added the LSP1 data keys in `LSP1Delegate.md` from the Contracts section.
- Added the Type IDs in `LSP1Delegate.md` from the Contracts section.
- Added details about LSP1 Routing in `LSP1Delegate.md` from the Contracts section.

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-8626 Improve LSP1 docs](https://app.clickup.com/t/2645698/DEV-8626)

**Comment by Hugoo** [comment]
Task linked: [DEV-8626 Improve LSP1 docs](https://app.clickup.com/t/2645698/DEV-8626)

**Comment by Hugoo** [comment]
Task linked: [DEV-8626 Improve LSP1 docs](https://app.clickup.com/t/2645698/DEV-8626)

**Comment by Hugoo** [comment]
Task linked: [DEV-8626 Improve LSP1 docs](https://app.clickup.com/t/2645698/DEV-8626)

File: docs/contracts/overview/LSP1Delegate.md [hunk 1]
```diff
@@ -10,3 +10,12 @@ There are two default implementations of the LSP1 Delegate contracts available,
 | ----------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------- |
 | **[`LSP1UniversalReceiverDelegateUP.sol`](../contracts/LSP1UniversalReceiver/LSP1UniversalReceiverDelegateUP/LSP1UniversalReceiverDelegateUP.md)**          | Add feature to a `LSP0ERC725Account` to registered and un-register tokens (LSP7 and LSP8) and vaults (LSP9) received. |
 | **[`LSP1UniversalReceiverDelegateVault.sol`](../contracts/LSP1UniversalReceiver/LSP1UniversalReceiverDelegateVault/LSP1UniversalReceiverDelegateVault.md)** | Add feature to a `LSP9Vault` to register and un-register tokens (LSP7 and LSP8).                                      |
+
+
+## Routing
+
+The Universal Profile implementation allows for attaching multiple instances of Universal Receiver Delegate to it. This is due to LSP1 Routing. You could have a signle General Purpose Unviersal Receiver Delegate and multiple instances of Specific Unviersal Receiver Delegate.
```
Comment (YamenMerhi) [inline docs/contracts/overview/LSP1Delegate.md:17]:
```suggestion
The Universal Profile implementation allows for attaching multiple instances of Universal Receiver Delegate to it. This is due to LSP1 Routing. You could have a single General Purpose Unviersal Receiver Delegate and multiple instances of Specific Unviersal Receiver Delegate.
```

File: docs/contracts/overview/LSP1Delegate.md [hunk 2]
```diff
@@ -10,3 +10,12 @@ There are two default implementations of the LSP1 Delegate contracts available,
...
+
+A General Purpose Universal Receiver Delegate has its address stored under the [LSP1UniversalReceiverDelegate Singleton](../../standards//generic-standards/lsp1-universal-receiver-delegate.md#lsp1universalreceiverdelegate-singleton) data key. This contract will be called on any LSP1 hook, meaning that it can potentially react on any [TypeID](./type-ids#universal-receiver-type-ids).
+
+A Specific Universal Receiver Delegate has its address stored under the [LSP1UniversalReceiverDelegate Singleton](../../standards//generic-standards/lsp1-universal-receiver-delegate.md#lsp1universalreceiverdelegate-mapping) data key. This contract will be called on specific LSP1 hook, depending on the [TypeID](./type-ids#universal-receiver-type-ids) that was passed as a parameter in the hook, meaning that it only reacts on a signle specific [TypeID](./type-ids#universal-receiver-type-ids).
```
Comment (YamenMerhi) [inline docs/contracts/overview/LSP1Delegate.md:21]:
```suggestion
A Specific Universal Receiver Delegate has its address stored under the [LSP1UniversalReceiverDelegate Mapping](../../standards//generic-standards/lsp1-universal-receiver-delegate.md#lsp1universalreceiverdelegate-mapping) data key. This contract will be called on a specific LSP1 hook, depending on the [TypeID](./type-ids#universal-receiver-type-ids) that was passed as a parameter in the hook, meaning that it only reacts on a single specific [TypeID](./type-ids#universal-receiver-type-ids).
```

**Review by CJ42 — APPROVED** [review]

**Pull request merged**